plugins/package-managers/node/src/main/kotlin/pnpm/PnpmDependencyHandler.kt

```diff
@@ -69,7 +69,7 @@ internal class PnpmDependencyHandler(private val pnpm: Pnpm) : DependencyHandler
         dependency.takeUnless { it.isProject() || !it.isInstalled }?.let {
             parsePackage(
                 packageJsonFile = it.packageJsonFile,
-                getRemotePackageDetails = pnpm::getRemotePackageDetails
+                getPackageDetails = pnpm::getRemotePackageDetails
```

Comment:
Should this `getRemotePackageDetails` function be renamed for the same reason?

Reply:
No, because the PNPM implementation indeed queries a remote (unless the result is cached).

Reply:
One could consider the "Remote" as implementation detail and strip it.

Don't do all of these function get the details potentially over the network?

Reply:
> Don't do all of these function get the details potentially over the network?

Yes, that's why I believe it's fine and good to include "Remote" in the name of those actual function implementations. However, `parsePackage()` does not care whether the passed details come from a remote or from elsewhere, so I'm omitting "Remote" from the parameter name.